app/controllers/idv/confirmations_controller.rb

```diff
@@ -19,16 +19,10 @@ def update
     end
 
     def download
-      personal_key = user_session[:personal_key]
+      code = personal_key
 
-      analytics.track_event(Analytics::IDV_DOWNLOAD_PERSONAL_KEY, success: personal_key.present?)
-
-      if personal_key.present?
-        data = personal_key + "\r\n"
-        send_data data, filename: 'personal_key.txt'
-      else
-        head :bad_request
-      end
+      analytics.track_event(Analytics::IDV_DOWNLOAD_PERSONAL_KEY, success: code.present?)
```

Comment:
This logging was introduced in #5215, I assume to be able to find the sorts of issues we were seeing then. If we're addressing that and assuming code will always be set, do we still need the logging? Or at least the `success` property?

Reply:
We still want the event to help us track when people do download, but I guess now the `success` would help us detect some not-expected degenerate cases where random key generation fails